docs/user-guide/102-policy-syntax.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Policy Syntax"
+tag: "user-guide"
+---
+
+The policy contains a pair of fields `capture` field to specify some simple expressions
```

Comment:
I'm having hard time following this paragraph, could you simplify this a bit?
What do you mean by `a pair of fields capture`? I only see one capture field containing expressions.

Reply:
I have split it in a pair of bullets.